package/yast2-online-update.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Jan  8 07:59:39 UTC 2015 - lslezak@suse.cz
+
+- Reporting products that have reached their end of support when
+  Online Update module is started (FATE#300591)
+- 2.17.24
```

Comment:
That's a generic problem for all Yast packages, you need to be careful when changing version in a maintenance update...